app/src/main/java/org/kiwix/kiwixmobile/main/KiwixMainActivity.kt

```diff
@@ -105,6 +105,10 @@ class KiwixMainActivity : CoreMainActivity() {
     kiwixActivityComponent.inject(this)
   }
 
```

Comment:
Kiwix/Custom/CoreMain should be shells of themselves after all their functionality is put into fragments. I'd want to see these deletions in this PR